src/activitypub/fediverse-bridge.ts

```diff
@@ -36,6 +47,88 @@ export class FediverseBridge {
         );
     }
 
+    private async handlePostCreated(event: PostCreatedEvent) {
+        const post = event.getPost();
+        if (!post.author.isInternal) {
+            return;
+        }
+        const ctx = this.fedifyContextFactory.getFedifyContext();
+        let fedifyObject = null;
+
+        if (post.type === PostType.Note) {
+            fedifyObject = new FedifyNote({
+                id: post.apId || ctx.getObjectUri(FedifyNote, { id: uuidv4() }),
```

Comment:
When is `post.apId` not defined?

Reply:
Fixed in https://github.com/TryGhost/ActivityPub/pull/551/commits/4a4a080f43440366183d9ed983a77132f8a21e9a